script/setup-mac-for-dev.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+export CHECK_VERSION=$1
+
+which -s brew
+if [[ $? != 0 ]] ; then
+    # Install Homebrew
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+else
+    brew update
+fi
+
+which -s python2
+if [[ $? != 0 ]] ; then
+    # Install python2
+    brew install python2
+fi
+
+which -s python3
+if [[ $? != 0 ]] ; then
+    # Install python3
+    brew install python3
+fi
+
+if [ ! -f /usr/local/bin/gtar ]; then
+	brew install gnu-tar
+	ln -fs /usr/local/bin/gtar /usr/local/bin/tar
+fi
+
+XCODE_VERSION=$(curl -v https://raw.githubusercontent.com/electron/electron/${CHECK_VERSION}/.circleci/config.yml | grep "xcode:"  | head -n1 | awk '{print $2}' | tr -d '"')
```

Comment:
Why doesn't this use the locally checked out .circleci/config.yml file?  If you are working on an older version of Electron, it will most like be using an older version of xcode

Reply:
My initial idea was to let the user download the script alone and pass all the needed arguments to the `verify-reproducibility.sh` script. That alone would suffice to actually download and verify electron.

That would be enough to bootstrap all the process from a single script. Do you think is better to download the whole repository before instead?

Reply:
ok that makes sense about downloading the script, but that means that it wouldn't be helpful for older versions of Electron.

Reply:
No it won't for macOS unfortunately. It should instead work for Linux. Do you think is better to embed the setup phase inside the same script for macOS too, to have it self contained?

Reply:
No it won't for macOS unfortunately. It should instead work for Linux. Do you think is better to embed the setup phase inside the same script for macOS too, to have it self contained?

Reply:
I guess its fine to leave the scripts as is.